[2021-08-12 10:34:33,030] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2021-01-01T00:00:00+00:00 [queued]>
[2021-08-12 10:34:33,036] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2021-01-01T00:00:00+00:00 [queued]>
[2021-08-12 10:34:33,036] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 10:34:33,036] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-12 10:34:33,036] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 10:34:33,043] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_2> on 2021-01-01T00:00:00+00:00
[2021-08-12 10:34:33,048] {standard_task_runner.py:52} INFO - Started process 129893 to run task
[2021-08-12 10:34:33,054] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_2', '2021-01-01T00:00:00+00:00', '--job-id', '31', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/subdag_example.py', '--cfg-path', '/tmp/tmpexw7a6v1', '--error-file', '/tmp/tmpnm1x3_x8']
[2021-08-12 10:34:33,054] {standard_task_runner.py:77} INFO - Job 31: Subtask task_2
[2021-08-12 10:34:33,084] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_2 2021-01-01T00:00:00+00:00 [running]> on host adam
[2021-08-12 10:34:33,114] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2021-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-01T00:00:00+00:00
[2021-08-12 10:34:33,115] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-12 10:34:33,115] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo task 2']
[2021-08-12 10:34:33,120] {subprocess.py:74} INFO - Output:
[2021-08-12 10:34:33,121] {subprocess.py:78} INFO - task 2
[2021-08-12 10:34:33,121] {subprocess.py:82} INFO - Command exited with return code 0
[2021-08-12 10:34:33,135] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_2, execution_date=20210101T000000, start_date=20210812T083433, end_date=20210812T083433
[2021-08-12 10:34:33,151] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-12 10:34:33,185] {local_task_job.py:149} INFO - Task exited with return code 0
